### PR TITLE
Add changelog for v1.2.1 release

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR adds changelog for the v1.2.1 release.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by xing-yang** [comment]
@msau42 PTAL.

File: CHANGELOG-1.2.1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog since v1.2.0
```
Comment (msau42) [inline CHANGELOG-1.2.1.md]: Instead of adding a new file, can we prepend it to the top of the CHANGELOG-1.2.md file?

Comment (xing-yang) [reply CHANGELOG-1.2.1.md]: Sure

**Comment by xing-yang** [comment]
Prepend 1.2.1 chnages to the top of the CHANGELOG-1.2.md.

**Pull request merged**